app/views/automated_tests/_test_script_result.html.erb

```diff
@@ -7,9 +7,9 @@
       <%= l(test_script_result.created_at, format: :long) %>
     </span>
   </h3>
-
-  <% tests = test_script_output['testrun']['test_script']['test'] %>
+  <% tests_scripts_outputs = test_script_output['testrun']['test_script'] %>
```

Comment:
@alexvail Okay I looked into this more. I got an error when you reverted the change, and I'm not sure why I wasn't getting that error before. In any case, I put in a simple fix, which you can see in @57edf2c15fa5eff458b61b966921ed20f2f3a460. We will talk later about handling the different levels of test results.